### PR TITLE
feat(sent-emails): Get all sent messages

### PR DESCRIPTION
**Pull request by amoskeyz**
#### What does this PR do?
Gets all sent message

#### Description of Task to be completed?
-Have the "GET '/api/v1/message/sent' " endpoint working
-update sent message controller
-test endpoint

#### How should this be manually tested?
-After cloning the repo, cd into it run "npm install" and run "npm run startdev",
-Using postman, test the endpoint
-if successful, you'll get a response with status code of 200
-run "npm run test" to test Api endpoint

#### What are the relevant pivotal tracker stories?
<a href='https://www.pivotaltracker.com/story/show/164787660'>#164787660</a>

### CODE REVIEW

File: Server/controllers/messageController.js [hunk 1]
```diff
@@ -26,14 +26,30 @@ class messageController {
...
-      data: array,
-    });
+  static async sentMessages(req, res) {
+    const id = req.decoder;
+    try {
+      const response = await pool.query('SELECT createdon, subject, message, parentmessageid, status FROM messages where senderid = $1', [id]);
+      console.log(response.rows[0]);
+      if (response.rows[0] === undefined) {
+        return res.status(400).json({
+          status: 'Unsuccessful',
```
Comment (omobosteven) [inline Server/controllers/messageController.js]: change the value of the status to `error`

File: Server/controllers/userController.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ class userController {
         });
       }
     } catch (err) {
+      console.log(err);
```
Comment (omobosteven) [inline Server/controllers/userController.js]: remove this

File: Server/controllers/messageController.js (hunk 1 above)
Comment (amoskeyz) [reply Server/controllers/messageController.js]: done

File: Server/controllers/userController.js (hunk 2 above)
Comment (amoskeyz) [reply Server/controllers/userController.js]: done

**Pull request merged**